### DISCUSSION TITLE
Request clean build

### DISCUSSION THREAD

**Discussion by WAR-DEPT** [ROM-builders/temporary#23641] [General]
https://cirrus-ci.com/task/5672035255320576?logs=build#L229

https://cirrus-ci.com/task/5351350800744448

What is this error ? Pls help 🙏

**Comment by Apon77** [comment]
I compiled locally with the same local manifest. No need to rerun.
Download link https://sourceforge.net/projects/rom-builders/files/miatoll/lineage-20.0-20230825-UNOFFICIAL-miatoll.zip/download
